daemon/daemon_unix.go

```diff
@@ -1190,6 +1191,15 @@ func selectBIP(
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "list bridge addresses failed")
 	}
+	// For IPv6, ignore the kernel-assigned link-local address. Remove all
+	// link-local addresses, unless fixed-cidr-v6 has the standard link-local
+	// prefix. (If fixed-cidr-v6 is the standard LL prefix, the kernel-assigned
+	// address is likely to be used instead of an IPAM assigned address.)
+	if family == netlink.FAMILY_V6 && (fCidrIP == nil || !isStandardLL(fCidrIP)) {
```

Comment:
Nit (not need to fix); looks like the `fCidrIP == nil` is redundant here, because  `isStandardLL` already returns false if it's `nil`